tempesta_fw/t/functional/tests/bomber.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+__author__ = 'NatSys Lab'
+__copyright__ = 'Copyright (C) 2016 NatSys Lab. (info@natsys-lab.com).'
+__license__ = 'GPL2'
+
+
+import conf
+import tfw
+
+c = conf.Config("etc/tempesta_fw.conf")
```

Comment:
It seems the config should be placed somewhere like ```t/functional/etc```. Or just, preferably, be generated from scratch in /tmp for each test.